samples/11.qnamaker/src/main/java/com/microsoft/bot/sample/qnamaker/QnABot.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.qnamaker;
+
+import com.microsoft.bot.ai.qna.QnAMaker;
+import com.microsoft.bot.ai.qna.QnAMakerEndpoint;
+import com.microsoft.bot.ai.qna.QnAMakerOptions;
+import com.microsoft.bot.builder.ActivityHandler;
+import com.microsoft.bot.builder.MessageFactory;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.integration.Configuration;
+import org.slf4j.LoggerFactory;
+import java.util.concurrent.CompletableFuture;
+
+public class QnABot extends ActivityHandler {
```

Comment:
Can we move the entire class into a `bots` folder just like [C#](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/11.qnamaker/Bots)?

Reply:
I didn't see the class being inside a bots folder in [multilingual java](https://github.com/microsoft/botbuilder-java/tree/main/samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual) or the other samples 🤔. I will move it though, np

Reply:
Ack, my bad. I double checked and you are right. Don't change this

Reply:
yikes, this never refreshed :(. Guess I can go to a previous commit